x-pack/metricbeat/module/azure/database_account/_meta/data.json

```diff
@@ -0,0 +1,40 @@
+{
+    "@timestamp" : "2020-02-25T08:53:00.000Z",
+    "cloud" : {
+        "provider" : "azure",
+        "region" : "westeurope"
+    },
+    "event" : {
+        "module" : "azure",
+        "duration" : 4877063600,
+        "dataset" : "azure.database_account"
+    },
+    "metricset" : {
+        "name" : "database_account",
+        "period" : 300000
+    },
+    "azure" : {
+        "dimensions" : {
+            "databasename" : "testdb"
+        },
+        "database_account" : {
+            "timegrain" : "PT5M",
+            "provisionedthroughput" : {
```

Comment:
Are we grouping metrics with the same dimensions & resource in a single document? That would save some space and make queries faster

Reply:
yes, all metrics with the same timegrain, dimension and resource are grouped in the same event